podman/domain/registry_data.py

```diff
@@ -40,7 +40,7 @@ def pull(self, platform: Optional[str] = None) -> Image:
         Args:
             platform: Platform for which to pull Image. Default: None (all platforms.)
         """
-        repository = api.parse_repository(self.image_name)
+        repository, _ = api.parse_repository(self.image_name)
```

Comment:
Not sure what is the usecase of this, just I found It is calling the helper ignoring that it returns a tuple.